packages/fast-components-react-msft/src/carousel/carousel.stories.tsx

```diff
@@ -3,163 +3,87 @@ import React from "react";
 import { Carousel, CarouselSlideTheme } from "./";
 import { uniqueId } from "lodash-es";
 import CarouselHero from "../../assets/carousel-hero-content";
+import { HeadingSize, ParagraphSize } from "../../src";
+
+function itemsFiller(
+    itemCount: number,
+    headerContent?: string,
+    paragraphContent?: string,
+    extraButtons?: boolean
+): any[] {
+    const fillerArray: any[] = [];
+    for (let i: number = 0, x: number = 1; i < itemCount; i++, x *= -1) {
+        const theme: any = x > 0 ? CarouselSlideTheme.light : CarouselSlideTheme.dark;
+        fillerArray.push({
+            id: uniqueId(),
+            theme,
+            content: (className?: string): React.ReactNode => (
+                <CarouselHero
+                    className={className}
+                    heading={
+                        headerContent
+                            ? {
+                                  size: HeadingSize._5,
+                                  children: headerContent,
+                              }
+                            : { children: `Slide ${i + 1}` }
+                    }
+                    paragraph={
+                        paragraphContent
+                            ? {
+                                  children: paragraphContent,
+                                  size: ParagraphSize._3,
+                              }
+                            : undefined
+                    }
+                    theme={theme}
+                    extraButtons={extraButtons}
+                />
+            ),
+        });
+    }
+    return fillerArray;
+}
+
```

Comment:
Can you ensure the types are accurate here? A number of `any` types.

Reply:
Yeah, sorry spaced on that.